sql/2020/04_Fonts/04_11a.popular_typeface.sql

```diff
@@ -1,32 +1,48 @@
 #standardSQL
 #popular_typeface
 CREATE TEMPORARY FUNCTION getFontFamilies(css STRING)
-RETURNS ARRAY < STRING > LANGUAGE js AS '''
+RETURNS ARRAY <STRING> LANGUAGE js AS '''
 try {
-    var $ = JSON.parse(css);
-    return $.stylesheet.rules.filter(rule => rule.type == 'font-face').map(rule => {
-        var family = rule.declarations && rule.declarations.find(d => d.property == 'font-family');
-        return family && family.value.replace(/[\'"]/g, '');
-    }).filter(family => family);
+  var $ = JSON.parse(css);
+  return $.stylesheet.rules.filter(rule => rule.type == 'font-face').map(rule => {
+    var family = rule.declarations && rule.declarations.find(d => d.property == 'font-family');
+    return family && family.value.replace(/[\'"]/g, '');
+  }).filter(family => family);
 } catch (e) {
-    return [];
+  return [];
 }
 ''';
+
 SELECT
   client,
   font_family,
-  COUNT(0) AS freq,
-  SUM(COUNT(0)) OVER (PARTITION BY client) AS total,
-  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY client) AS pct
-FROM
-  `httparchive.almanac.parsed_css`,
-  UNNEST(getFontFamilies(css)) AS font_family
+  pages,
+  total,
+  pages / total AS pct
+FROM (
+  SELECT
+    client,
+    font_family,
+    COUNT(DISTINCT page) AS pages
+  FROM
+    `httparchive.almanac.parsed_css`,
+    UNNEST(getFontFamilies(css)) AS font_family
+  WHERE
+    date = '2020-08-01'
+  GROUP BY
+    client,
+    font_family)
+JOIN (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    COUNT(0) AS total
+  FROM
+    `httparchive.summary_pages.2020_08_01_*`
+  GROUP BY
+    client)
+USING
+  (client)
 WHERE
-  date="2020-08-01"
-GROUP BY
-  client,
-  font_family
-HAVING
-  freq >= 1000 
+  pages / total >= 0.01
```

Comment:
request a slight change as like to see Noto Sans JP (0.45%) in list.
```suggestion
  pages / total >= 0.004
```